### DISCUSSION TITLE
Search not working

### DISCUSSION THREAD

**Discussion by vqoch** [valinet/ExplorerPatcher#1567] [Q&A] [answered]
Since the last windows update if I bring up the start menu and start typing away, instead of it searching for the said keystrokes, it just cycles through the pinned items.

Clicking on the search box or hitting Win+S does nothing either.

I remember having this problem before and finding a fix on here, but I can't remember what it was.

Can anyone help?

**Comment by pyrates999** [comment]
https://github.com/valinet/ExplorerPatcher/issues/1109

**Comment by vqoch** [comment, marked as answer]
A simple computer restart fixed the issue.,
